src/components/movies/MovieCard.vue

```diff
@@ -0,0 +1,80 @@
+<template>
+    <div class="movie__card" v-if="movie">
+        <h1 class="movie__title"> {{ movieTitle }}</h1>
+        <div class="movie__info">
+            <p class="movie__duration-time">{{ movieLength }}</p>
+            <img class="movie__poster" :src="moviePoster" />
+            <div class="movie__category">
+                {{ movieCategory }}
+            </div>
+        </div>
+    </div>
+</template>
+
+<script>
+export default {
+    props: {
+        movie: {
+            type: Object
+        }
+    },
+    methods: {
+        formateTime(movieLength) {
+            const hours = Math.floor(movieLength / 60);  
+            const minutes = String('0' + (movieLength % 60)).slice(-2) + ' min'
+            return hours + ":" + minutes
+        }
+    },
+    computed: {
+        movieTitle() {
+            return this.movie.title
```

Comment:
`movie` is not required so potentially it may be undefined

Reply:
You'r right, but on top of the component I do check if there is movie so isn't it enough?

Reply:
I believe if we finally get proper movie object it will be complete but to be sure there will be no error in any case i opt to use optional chaining operator (.?) wdyt?

Reply:
As you said if the computed value won't be used (e.g. because of `v-if`) there won't be any error. However, it is not a good practice to have a computed that can e used only under certain conditions - you should always be safe when just reading a computed value 😉 

Chaining operator would be great 👌